usd/src/usd_parser/USD2SDF.cc

```diff
@@ -75,6 +81,96 @@ UsdErrors USD2SDF::Read(const std::string &_filename,
   return errors;
 }
 
+////////////////////////////////////////////////////////////////////////////////
+void USD2SDF::AddLights(
```

Comment:
I'm confused about the role of this function.
* It looks like it is modifying the xml directly, shouldn't we use sdf apis instead?
* It looks like usd is already parsed into sdf by this point, why do we need this function?